libkbfs/init.go

```diff
@@ -486,8 +486,11 @@ func Init(ctx Context, params InitParams, keybaseServiceCn KeybaseServiceCn, onI
 	// -mdserver point to local implementations.
 
 	if len(params.WriteJournalRoot) > 0 {
-		config.EnableJournaling(params.WriteJournalRoot,
+		err := config.EnableJournaling(params.WriteJournalRoot,
 			params.TLFJournalBackgroundWorkStatus)
+		if err != nil {
+			return nil, err
```

Comment:
Did you fix the race preventing journaling from coming up sometimes?  I'm worried that if the race is still there, this will break KBFS for existing users in hard-to-discover ways, even before journaling is on by default.